src/one_dragon/base/config/user_config.py

```
@@ -0,0 +1,231 @@
+            text = self._to_json_text(data)
+            SQLITE_OPERATOR.save(self.key, text)
+        except Exception:
+            log.error("配置迁移写入数据库失败 %s", file_path, exc_info=True)
+            return None
+
+        # 迁移完成后删除原文件并清理空目录
+        self._cleanup_source_file_and_dirs(file_path)
+        return text
+
+    def _cleanup_source_file_and_dirs(self, path: Path):
+        """删除源 yml 文件，并自下而上清理空目录直到 config 根目录。"""
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)